os/deploy_coreos_libvirt.sh

```diff
@@ -38,12 +39,12 @@ for SEQ in $(seq 1 $1); do
                 mkdir -p $LIBVIRT_PATH/$COREOS_HOSTNAME/openstack/latest || (echo "Can not create $LIBVIRT_PATH/$COREOS_HOSTNAME/openstack/latest directory" && exit 1)
         fi
 
-        if [ ! -f $LIBVIRT_PATH/coreos_production_qemu_image.img ]; then
-                wget http://stable.release.core-os.net/amd64-usr/current/coreos_production_qemu_image.img.bz2 -O - | bzcat > $LIBVIRT_PATH/coreos_production_qemu_image.img
+        if [ ! -f $LIBVIRT_PATH/coreos_${CHANNEL}_qemu_image.img ]; then
```

Comment:
It doesn't matter. I use pipe and "bzcat > $LIBVIRT_PATH/coreos_${CHANNEL}_qemu_image.img"